src/Microsoft.Framework.ApplicationHost2/Program.cs

```diff
@@ -22,9 +26,112 @@ public Program(IAssemblyLoaderContainer container, IApplicationEnvironment envir
 
         public Task<int> Main(string[] args)
         {
-            Console.WriteLine("I AM APP HOST 2!");
-            Console.WriteLine("TODO: Actually do things!");
+            ApplicationHostOptions options;
+            string[] programArgs;
```

Comment:
It is for future use. I'm trying to update with code as I go so we don't end up with one giant review, but that may be more trouble than it's worth :)